packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -23,7 +23,7 @@ const { details: X, quote: q } = assert;
 const trace = makeTracer('IV');
 
 /**
- * @file This has most of the logic for a Vault, to borrow RUN against collateral.
+ * @file This Has most of the logic for a Vault, to borrow RUN against collateral.
```

Comment:
I don't think this `@file` take a keyword before the prose

Reply:
this was fixed 10 hours ago, will test https://github.com/hosseinmd/prettier-plugin-jsdoc/issues/143#issuecomment-1077458920